src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -7,6 +7,7 @@ import akka.stream.ActorMaterializer
 import mouse.any._
 import com.typesafe.scalalogging.LazyLogging
 import org.slf4j.LoggerFactory
+import slick.lifted.{AbstractTable, TableQuery}
```

Comment:
these imports might not be needed if the code for initTableFromCsv gets simplified

Reply:
no, they are needed